autoload/firenvim.vim

```diff
@@ -895,6 +895,38 @@ function! firenvim#install(...) abort
                 endif
         endif
 
+        try
+                if !s:is_wsl
+                        call firenvim#internal_install(l:force_install, l:script_prolog)
+                        let s:is_wsl = has('wsl') || !empty($WSL_DISTRO_NAME) || !empty ($WSL_INTEROP)
+                        if s:is_wsl
+                                echo 'Installation complete on the wsl side. Performing install on the windows side.'
+                                call firenvim#internal_install(l:force_install, l:script_prolog)
+                        endif
+                else
+                        throw 'firenvim#install is called, but s:is_wsl has not been reset. Please report this bug.'
+                endif
+        finally
+                if s:is_wsl
+                        let s:is_wsl = v:false
+                endif
```

Comment:
```suggestion
                let s:is_wsl = v:false
```
What do you think about doing this instead? (And likewise in `firenvim#uninstall`)

Reply:
I think that should be cleaner. The code is updated.